src/database/datatools/scheduletypelimits.py

```
@@ -0,0 +1,135 @@
+        updated_numeric_type, 
+        updated_unit_type
+    ]
+    update_description_schedule_type_limits(db_path, des_data) 
+
+def delete_scheduletypelimits(db_path: str, scheduletypelimits_id: int) -> None:
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+
+    table_name = "schedule_type_limits"
+
+    sql = f"DELETE FROM {table_name} WHERE id = ?"
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)